dub.sdl

```diff
@@ -20,9 +20,11 @@ buildType "unittest-release" {
 	versions "mir_test"
 }
 
+dflags "-preview=dip1008"
+
```

Comment:
was this deliberate? It breaks anyone who can't use dip1008.

Also, dub doesn't propagate dflags to the whole build, only to packages that depend on the package that sets them, which means you end up building part of the build with dip1008 and part without. I noticed because linker errors where something was `@nogc` in the root package but not in its dependencies. So in general it just doesn't work to set dip flags in a library config.

Reply:
No. Interesting is there a case when the DIP can't be used? I thought it is quite good.

I will create a patch tomorrow.

Reply:
To be honest, there probably aren't many practical cases where you can't use it (maybe if you catch and store exceptions?), but nonetheless it does cause quite confusing breakage (because dub and dflags don't work well)